bin/gwd/request.ml

```diff
@@ -552,6 +552,18 @@ let treat_request =
           w_wizard @@ w_base @@ UpdateInd.print_del
         | "DEL_IND_OK" ->
           w_wizard @@ w_lock @@ w_base @@ UpdateIndOk.print_del
+        | "DOC" ->
+          w_base @@ fun conf base -> (
+          match Util.p_getenv conf.env "s" with
+          | Some f ->
+                if Filename.check_suffix f ".txt" then
+                  let f = Filename.chop_suffix f ".txt" in
+                  SrcfileDisplay.print_source conf base f
+                else (
+                 match Util.p_getenv conf.env "t" with
```

Comment:
and this "t"?

Reply:
This is to mimic the m=IM/m=IMH existing solution: 
m=DOC/m=DOC&t=H
Could also be done with a m=DOCH command.